lib/node_modules/@stdlib/assert/is-same-accessor-array/docs/types/test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import isSameAccessorArray = require( './index' );
+
+
+// TESTS //
+
+// The function returns a boolean...
+{
+	isSameAccessorArray( 3.14, 3.14 ); // $ExpectType boolean
```

Comment:
@kgryte We should go ahead and update the `types/test.ts` files for all such `assert` packages, where we never test the cases where they would return `true`.

Reply:
Sure. Feel free to go ahead and make those changes.